### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by IgorSusmelj**
## Changes

- fix a few typos in the readme
- switchto show the new low-level building blocks on ho to use lightly
- remove section about CLI usage
- add papers from research using lightly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,10 +89,10 @@ We strongly recommend that you install Lightly in a dedicated virtualenv, to avo
 
 ### Lightly in Action
 
-With lightly you can use latest self-supervised learning methods in a modular
+With lightly, you can use the latest self-supervised learning methods in a modular
```
Comment (japrescott) [inline README.md:92]: Lightly big, no?

File: README.md [hunk 2]
```diff
@@ -107,14 +114,27 @@ dataloader = torch.utils.data.DataLoader(
     shuffle=True,           # shuffling is important!
     collate_fn=collate_fn)  # apply transformations to the input images
 
+
+# create a PyTorch module for the SimCLR model
+class SimCLR(nn.Module):
+    def __init__(self, backbone):
+        super().__init__()
+        self.backbone = backbone
+        self.projection_head = models.modules.SimCLRProjectionHead(512, 512, 128)
```
Comment (japrescott) [inline README.md:123]: maybe you can explain the 512,512,128 a bit better unless thats clear for all pros

File: README.md [hunk 3]
```diff
@@ -224,7 +244,7 @@ Below you can see a schematic overview of the different concepts present in the
 
 
 ### Next Steps
-Head to the [documentation](https://docs.lightly.ai) and see the things you can achieve with Lightly!
+Head to the [documentation](https://docs.lightly.ai) and see the things you can achieve with lightly!
```
Comment (japrescott) [inline README.md:247]: Lightly big?

File: README.md [hunk 4]
```diff
@@ -239,9 +259,9 @@ For more information about how to contribute have a look [here](CONTRIBUTING.md)
 
 ### Running Tests
 
-Unit tests are within the `tests` folder and we recommend to run them using 
+Unit tests are within the `tests` folder and we recommend running them using
```
Comment (japrescott) [inline README.md:262]: maybe link `tests` to the folder?

File: README.md [hunk 5]
```diff
@@ -287,11 +307,20 @@ pylint lightly/core.py
   we need to make training deep learning models more data efficient to achieve widespread adoption. One step to achieve this goal is by leveraging self-supervised learning. The company behind lightly commited to keep this framework open-source.
 
 - If this framework is free, how is the company behind lightly making money?
-  - Training self-supervised models is only part of the solution. The company behind lightly focuses on processing and analyzing embeddings created by self-supervised models. 
+  - Training self-supervised models is only part of our solution. The company behind lightly focuses on processing and analyzing embeddings created by self-supervised models.
```
Comment (japrescott) [inline README.md:310]: is only part of our solution -> is only one part of our solution

**Review by japrescott — COMMENTED** [review]
Looks great. Some suggestions added

File: README.md (hunk 1 above)
Comment (IgorSusmelj) [reply README.md:92]: We said we would keep everything OSS related lowercase

File: README.md (hunk 3 above)
Comment (IgorSusmelj) [reply README.md:247]: Would also keep this lower as it's OSS related

File: README.md (hunk 1 above)
Comment (japrescott) [reply README.md:92]: okeydokey

**Review by japrescott — APPROVED** [review]
can't really verify the code but lgtm

**Review by japrescott — APPROVED** [review]

**Pull request merged**